analyzer.go

```diff
@@ -252,6 +257,41 @@ func (a *Analyzer) getImageIdentifier(image imgutil.Image) (string, error) {
 	return identifier.String(), nil
 }
 
+func (a *Analyzer) getTargetFromImage(image imgutil.Image) (*platform.TargetMetadata, error) {
```

Comment:
I need to use this in the restorer also, so could we move this to the platform package?

Reply:
✅